server/routes/serviceProviderReferrals/serviceUserBannerPresenter.ts

```diff
@@ -0,0 +1,40 @@
+import { DeliusServiceUser } from '../../services/communityApiService'
+import ViewUtils from '../../utils/viewUtils'
+import utils from '../../utils/utils'
+
+export default class ServiceUserBannerPresenter {
+  constructor(private readonly serviceUser: DeliusServiceUser) {}
+
+  readonly serviceUserBannerArgs = {
+    titleText: 'Service user details',
+    html:
```

Comment:
you raised concerns about this HTML but since this a direct copy paste i want to do that refactoring seperately

Reply:
Makes sense. Could you make a tech debt issue for it in Jira so that we remember?

Reply:
done